famcial-service-api/src/main/java/com/zhenkangyao/famcial/models/AccountType.java

```diff
@@ -0,0 +1,8 @@
+package com.zhenkangyao.famcial.models;
```

Comment:
Is it good to add enums on models file? Or should we create new fold for enums? I don't know, we should Google this.